.golangci.yml

```
@@ -1,62 +1,63 @@
-# Please refer to the official golangci-lint config documentation for more details:
-# https://golangci-lint.run/usage/configuration/
-# https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml
+version: "2"
 run:
-  timeout: 10m
   tests: true
-  # default is true. Enables skipping of directories:
-  #   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
-  exclude-dirs-use-default: true
-
 linters:
-  disable-all: true
+  default: none
   enable:
-    - goimports
-    - gosimple
+    - bidichk
+    - copyloopvar
+    - durationcheck
+    - gosec
     - govet
     - ineffassign
     - misspell
+    - revive
+    - staticcheck
```

Would this make sense ?
```suggestion
    - revive
    - staticcheck
    - gosimple
    - goimports
```

---

`gosimple`:

```
The command is terminated due to an error: unknown linters: 'gosimple', run 'golangci-lint help linters' to see the list of supported linters
```

`goimports`:

```
Error: can't load config: goimports is a formatter
The command is terminated due to an error: can't load config: goimports is a formatter
make: *** [lint] Error 3
```

:point_up: This is already defined under the formatters